Microsoft.Azure.Cosmos/src/Pagination/CrossPartitionRangePageAsyncEnumerator.cs

```diff
@@ -177,7 +177,7 @@ public async ValueTask<bool> MoveNextAsync(ITrace trace)
                                 this.cancellationToken);
                         }
 
-                        if (childRanges.Count() <= 1)
+                        if (childRanges.Count() < 1)
```

Comment:
nit: childRanges is a list so just use the Count property. Please do this to the rest of places too.

```suggestion
                        if (childRanges.Count < 1)
```